site/src/components/onslogo.js

```diff
@@ -16,6 +16,7 @@ export default () => {
         css={css`
           padding: 0px 15px 20px 15px;
         `}
+        data-test="ons-logo__header"
```

Comment:
Can you use semantic class names instead of data-test attributes?   Coordinate with dev to get the class names you need if not sufficient / specific enough for test needs.

Reply:
Following discussion having data-test helps decouple the styling from the testing.  React components also move away from using classes and if test hooked into the specifics of React components then tests would be brittle.   Cancel my comment above.